[TLV-2021] Add Snyk as Sponsor

shar1z opened the pull request:
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

glasnt commented:
Hi @shar1z, did you mean to add Snyk as a community sponsor even though this PR also introduces a `branding` sponsor level? I'll merge and let you adjust in a separate PR if you need to.

shar1z commented:
> Hi @shar1z, did you mean to add Snyk as a community sponsor even though this PR also introduces a `branding` sponsor level? I'll merge and let you adjust in a separate PR if you need to.

Hi @glasnt - really appreciate the merge.
Snyk are a Community Pillar - we've introduced a new sponsorship tier and are in the final throes of locking those down, so I added the tier (albeit the sponsors are yet to come).  Really appreciate you checking in on this.
